services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/scheduler.py

```diff
@@ -397,6 +397,9 @@ def endpoint(self) -> AnyHttpUrl:
     request_scheme: str = Field(
         ..., description="used when configuring the CORS options on the proxy"
     )
+    request_simcore_user_agent: str = Field(
+        ..., description="used as label to filter out from the metrics"
+    )
     proxy_service_name: str = Field(None, description="service name given to the proxy")
 
     product_name: Optional[str] = Field(
```

Comment:
why is this one optional? does that make sense?

Reply:
This is just wrong. Fixing it